data\json\npcs\PS_SHEEP

```diff
@@ -0,0 +1,199 @@
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_SHEEP_KNOW",
+    "dynamic_line": "Yes.  They hold the secret.  The world rests on a very delicate equilibrium and the key is to count them, at all times.  I keep a detailed log in my head… we may still be able to undo everything!  When the time is right, I'll publish a paper, reveal the secret to the broader scientific community and we'll work together to stop the cataclysm.",
+    "responses": [
+      { "text": "Oh, okay.", "topic": "TALK_SHEEP" },
+	  { "text": "You do realise that there is no scientific community to speak of anymore, right?", "topic": "TALK_SHEEP_SCIENCE" }
+      { "text": "Okay, I'm done. See ya.", "topic": "TALK_DONE" }
+    ]
+  },
+  
+     {
+    "type": "talk_topic",
+    "id": "TALK_SHEEP_SCIENCE",
+    "dynamic_line": "There is a Hub of world's greatest minds out there.  They told me to keep counting for now, and so I shall.  I do not intend to let them down, I must keep a very detailed log, in my head, so no one can take it from me.",
+    "responses": [
+      { "text": "How many sheep have you counted so far?", "topic": "TALK_SHEEP_MANY" },
+      { "text": "Okay, I'm done.", "topic": "TALK_DONE" }
+    ]
+  },
+  
+  {
+      "type": "talk_topic",
+    "id": "TALK_SHEEP_MANY",
+    "dynamic_line": "Sixty-eight white, eleven brown, eight black, three red, two pink and one blue.  See?  Very detailed.",
+    "responses": [
+      { "text": "That's… fascinating.", "topic": "TALK_SHEEP" },
+      { "text": "Okay, I'm done.", "topic": "TALK_DONE" }
+    ]
+  },
+  
+  {
+      "type": "talk_topic",
+    "id": "TALK_SHEEP_WHY",
+    "dynamic_line": "I'm counting sheep.  I'm also hunting and fishing around these parts.  But I never kill sheep.  I just count them.  Very important.  You can kill, I can't, but I must count.  Must.  Always.",
+    "responses": [
+      { "text": "Great.", "topic": "TALK_SHEEP" },
+      { "text": "Okay, I'm done.", "topic": "TALK_DONE" }
+    ]
+  },
+  
+  
+      {
+      "type": "talk_topic",
+    "id": "TALK_SHEEP_IDIOT",
+    "dynamic_line": "Stupid is as stupid does.  I guess you just can't grasp the secret.  I don't blame you, takes a titanic intellect to do so.",
+    "responses": [
+      { "text": "Of course.", "topic": "TALK_SHEEP" },
+	  { "text": "There are… no words.", "topic": "TALK_DONE" },
+	  { "text": "Uh, huh.", "topic": "TALK_SHEEP" },
+      { "text": "Okay, I'm done.", "topic": "TALK_DONE" }
+    ]
+  },
+  
+    {
+    "type": "talk_topic",
+    "id": "TALK_SHEEP_BYE",
+    "dynamic_line": "I'll be counting.",
+    "responses": [ { "text": "…", "topic": "TALK_DONE" } ]
+  },
+  
+  {
+    "id": "MISSION_SHEEP_GATHER",
+    "type": "mission_definition",
+    "name": "I need new bolts for my crossbow.",
+    "description": "Help the odd hunter by finding 50 steel bolts for his crossbow.",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 2,
+    "value": 800,
+    "item": "bolt_steel",
```

Comment:
These are the non-craftable aluminun broadhead bolts - they aren't steel bolts (despite the id).